server/mock/datastore_mock.go

```diff
@@ -268,7 +268,7 @@ type SearchTeamsFunc func(ctx context.Context, filter fleet.TeamFilter, matchQue
 
 type TeamEnrollSecretsFunc func(ctx context.Context, teamID uint) ([]*fleet.EnrollSecret, error)
 
-type LoadHostSoftwareFunc func(ctx context.Context, host *fleet.Host) error
+type LoadHostSoftwareFunc func(ctx context.Context, host *fleet.Host, includeCVEScores bool) error
```

Comment:
`includeCVEScores` -> `includeCVEMeta`?

Reply:
I kept this as `includeCVEScores` because we are not planning the return the published date form the API at this time.

Reply:
oks